net.veloren.veloren.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: net.veloren.veloren
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+command: veloren-voxygen
+finish-args:
+  - "--device=all"
+  - "--env=VELOREN_ASSETS=/app/share/veloren/assets"
+  - "--share=ipc"
+  - "--share=network"
+  - "--socket=pulseaudio"
+  - "--socket=wayland"
+  - "--socket=x11"
+build-options:
+  append-path: /usr/lib/sdk/rust-nightly/bin
+  env:
+    RUSTFLAGS: --remap-path-prefix =../
+    CARGO_HOME: /run/build/veloren/cargo
+modules:
+  - name: veloren
+    buildsystem: simple
+    build-commands:
+      # Unoptimize dev/debug builds to speed up build process
+      #- sed -i 's/opt-level = 3/opt-level = 0/' Cargo.toml
+      #- sed -i 's/opt-level = 2/opt-level = 0/g' Cargo.toml
+
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      - install -D -m 755 ./target/release/veloren-{voxygen,{server,chat}-cli} -t /app/bin/
+    post-install:
+      # Install assets
+      - mkdir -p /app/share/veloren/
+      - cp -ap assets/ /app/share/veloren/
+
+      # Install desktop file, icon and AppData manifest
+      - install -D -m 644 assets/voxygen/net.veloren.veloren.metainfo.xml -t /app/share/metainfo/
+      - install -D -m 644 assets/voxygen/net.veloren.veloren.desktop -t /app/share/applications/
+      - install -D -m 644 assets/voxygen/net.veloren.veloren.png -t /app/share/icons/hicolor/256x256/apps/
+    sources:
+      - type: git
+        commit: b751fcc3937d8205cb2261c056aed7ffba218151
```

Comment:
```suggestion
        commit: 1dd1318ae01b0f76dee872440d599e2546402380
```

Reply:
It's a commit from not yet accepted [merged request](https://gitlab.com/veloren/veloren/-/merge_requests/907). Better use commit from master and add the MR as a patch, I guess.

Reply:
I can't accept this until [this MR](https://gitlab.com/veloren/veloren/-/merge_requests/907) will be merged.

Reply:
Or we can just apply this patch. This is just warning and there is just few commit differs in between. Online should work. Anyway we will wait for `0.6` tag i suppose before final merge. Offline mode works. We can't rebuild for every new commit, at least manually.

Reply:
mh.. ok sounds good. when i try to connect, after the warning, get these messages.
````
[WARN] Server is running 1dd1318a[2020-04-12-20:20], you are running b751fcc3[2020-04-12-22:14], versions might be incompatible!
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `1`, expected variant index 0 <= i < 1")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
BINCODE ERROR: Custom("invalid value: integer `13`, expected variant index 0 <= i < 2")
[ERROR] [session] Failed to tick the scene: ClientError(Network(Bincode(Custom("invalid value: integer `13`, expected variant index 0 <= i < 2"))))
```

Reply:
@frzifus fix coming. Let's automate a little build releases info in appdata and build version from master.